examples/03_connectivity/plot_multi_subject_connectome.py

```diff
@@ -101,11 +101,8 @@ def plot_matrices(cov, prec, title, labels):
 gsc = GroupSparseCovarianceCV(verbose=2)
 gsc.fit(subject_time_series)
 
-try:
-    from sklearn.covariance import GraphicalLassoCV
-except ImportError:
-    # for Scitkit-Learn < v0.20.0
-    from sklearn.covariance import GraphLassoCV as GraphicalLassoCV
```

Comment:
am I correct that this is not needed anymore given the scikit learn requirements we have?

Reply:
indeed.